# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Fix se_NO locale
# DP: Dpatch author: Denis Barbier
# DP: Patch author: Denis Barbier
# DP: Upstream status: Not submitted yet
# DP: Date: 2006-06-02

Index: b/localedata/locales/se_NO
===================================================================
--- a/localedata/locales/se_NO	2006-07-31 00:19:43.000000000 +0200
+++ b/localedata/locales/se_NO	2008-08-08 18:37:39.000000000 +0200
@@ -137,7 +137,7 @@
 <U01EF> <ezhcaron>;<CAR>;<MIN>;IGNORE % ezh caron
 
 reorder-after <U010A>
-<U01B7> <ezh>;<BAS>;<MIN>;IGNORE % EZH
+<U01B7> <ezh>;<BAS>;<CAP>;IGNORE % EZH
 <U01EE> <ezhcaron>;<CAR>;<CAP>;IGNORE % EZH caron
 
 reorder-after <U0064>
@@ -164,9 +164,10 @@
 reorder-after <U0136>
 <U01E8> <kcaron>;<CAR>;<CAP>;IGNORE % Kcaron
 
-reorder-after <U0053>
+reorder-after <U0073>
 <U00DF> <ssharp>;"<LIG><LIG>";"<MIN><MIN>";IGNORE % ß
 <U0161> <scaron>;<CAR>;<MIN>;IGNORE % 288 <s<>
+reorder-after <U0053>
 <U0160> <scaron>;<CAR>;<CAP>;IGNORE % 405 <S<>
 
 
